third_party/silabs/efr32_sdk.gni

```diff
@@ -244,7 +244,7 @@ template("efr32_sdk") {
         } else {
           defines += [ "LWIP_IPV4=0" ]
         }
-        if (lwip_ipv4) {
+        if (lwip_ipv6) {
```

Comment:
Since Matter is by default an IPV6 based protocol why make the IPV6 functionnality of LWIP optionnal ?

Reply:
Actually the IPV6 functionality should be hardcoded to 1 for ALL matter examples

Reply:
This is based on the `lwip_ipv6` flag which is available in `lwip.gni`, which includes the IPv6 sources if this flag is set, I just aligned with that. In future or anywhere if `lwip_ipv6` is set to false then adding `"LWIP_IPV6=1"` will not make any difference to LwIP. Since matter is providing it as an argument, I am using it as one. The flag `lwip_ipv6` is overwritten in the wifi_args.gni or similar file.

Reply:
Seems fair